src/queries/rum-pageviews-bdf.sql

```diff
@@ -0,0 +1,39 @@
+--- description: Show page views for specified domains for specified dates based on extrapolation from RUM data
+--- Authorization: none
+--- interval: 30
+--- offset: 0
+--- startdate: 2023-05-01
+--- enddate: 2023-06-01
+--- timezone: UTC
+--- url: -
+--- device: all
+--- domainkey: secret
+
+WITH rum AS (
+  SELECT
+    REGEXP_REPLACE(hostname, 'www.', '') AS hostname,
+    COUNT(DISTINCT id) AS rum_count,
+    weight,
+    FORMAT_DATE('%Y-%m-%d', time) AS date
+  FROM helix_rum.EVENTS_V3(
+    @url,
+    CAST(@offset AS INT64),
+    CAST(@interval AS INT64),
+    @startdate,
+    @enddate,
+    @timezone,
+    @device,
+    @domainkey
+  )
+  GROUP BY date, weight, hostname
+)
+SELECT
+  a.hostname,
+  SUM(a.rum_count * a.weight) AS estimated_pv,
+  a.date,
+  b.ims_org_id
+FROM rum a
```

Comment:
```suggestion
FROM rum AS rum_data
```

This makes it a bit easier to understand what's going on then `a` and `b`